frontend/src/pages/projects/typeHelpers.ts

```diff
@@ -0,0 +1,2 @@
+export type ValueOf<T> = T[keyof T];
```

Comment:
heh, meant the global one, not create a new file... but the 2nd item in this file probably could just be in a type file as it's custom functionality you're describing.